templates/page.tpl.php

```diff
@@ -89,16 +89,22 @@
 </main>
 
 <footer role="contentinfo">
-  <section class="page-footer">
     <div class="wrapper">
       <section class="footer-top">
         <nav>
-          <?php print render($page['footer']); ?>
+          <?php print render($page['footer_top']); ?>
         </nav>
       </section>
+      <section>
+        <div class="footer-logo"></div>
+        <div class="footer-links">
+          <nav>
+            <?php print render($page['footer']); ?>
+          </nav>
```

Comment:
I think we need to make this less specific in terms of content assumptions...

We could potentially sort this with two footer regions (top and bottom) and then have theme settings that dictated how they were displayed - horizontal or vertical.  This would affect how the blocks within those regions were laid out (regardless of the type of block).

This would allow us to support something like the [https://www.govcms.gov.au/](govCMS) footer where one region is inline and one is stacked.